lib/cartopy/mpl/geoaxes.py

```diff
@@ -1540,6 +1540,29 @@ def scatter(self, *args, **kwargs):
         self.autoscale_view()
         return result
 
+    @_add_transform
+    def hexbin(self, x, y, *args, **kwargs):
+        """
+        Add the "transform" keyword to :func:`~matplotlib.pyplot.hexbin'.
```

Comment:
In one of your earlier comments you mentioned that the data will always be projected to the map projection and hexes will always be drawn in the map projection (rather than the hexes themselves being transformed). Capturing that in the docstring would be good to let people know as well.